### PR TITLE
Minor ruby and gem bumps, nothing big or scary

### PR DESCRIPTION
**Pull request by denny**
Including Ruby 2.7.1 to 2.7.2 and Rails 6.0.3.3 to 6.0.3.4

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -174,13 +176,14 @@ GEM
...
       jwt (>= 2.0)
+      oauth2 (~> 1.1)
       omniauth (>= 1.1.1)
       omniauth-oauth2 (>= 1.6)
-    omniauth-oauth2 (1.6.0)
-      oauth2 (~> 1.1)
-      omniauth (~> 1.9)
+    omniauth-oauth2 (1.7.1)
+      oauth2 (~> 1.4)
+      omniauth (>= 1.9, < 3)
```
Comment (weatherpixie) [inline Gemfile.lock:186]: does this run into the same omniauth pain as everything else?

File: Gemfile.lock [hunk 2]
```diff
@@ -107,17 +106,20 @@ GEM
...
+    docile (1.3.5)
+    erubi (1.10.0)
     execjs (2.7.0)
-    faraday (1.0.1)
+    faraday (1.3.0)
+      faraday-net_http (~> 1.0)
       multipart-post (>= 1.2, < 3)
-    ffi (1.13.1)
+      ruby2_keywords
+    faraday-net_http (1.0.1)
```
Comment (weatherpixie) [inline Gemfile.lock:117]: should faraday-net_http be in twice?

**Review by weatherpixie — APPROVED** [review]

**Review by jkhulme — APPROVED** [review]

File: Gemfile.lock (hunk 2 above)
Comment (denny) [reply Gemfile.lock:117]: I think it's at two different levels; once in its own right, and once as a dependency of something else (`faraday`, in fact)

File: Gemfile.lock (hunk 1 above)
Comment (denny) [reply Gemfile.lock:186]: Oof, good question.

**Pull request merged**